Source/APIClient/Request.swift

```diff
@@ -18,32 +18,18 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
-    static var authToken: String? { get set }
+    static var authToken: String? { get }
```

Comment:
I think we only want to require that the `Request` protocol return an `authToken` value, not necessarily that it also have a settable token. In some cases, users may implement it to be settable as well, but it's not necessary for the base functionality of the protocol.